release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -1439,3 +1439,7 @@ $ oc get pods -n openshift-cluster-node-tuning-operator
 +
 Verify that the Pods are now in a `Running` state.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1791916[*BZ#1791916*])
+
+* The Node Feature Discovery (NFD) Operator version 4.2 is failing to deploy in
```

Comment:
version 4.3

Reply:
fails to deploy from OpertorHub on OpenShift Console.

As a workaround, download the oc client for your operating system, and place the kubeconfig from the installer in ~/.kube/config.  Follow these steps to deploy NFD Operator from CLI and github:

cd $GOPATH/src/openshift
git clone https://github.com/openshift/cluster-nfd-operator.git
cd cluster-nfd-operator
git checkout namespace-scoped
PULLPOLICY=Always make deploy
oc get pods -n openshift-nfd

Example:

# oc get pods -n openshift-nfd
NAME                           READY   STATUS    RESTARTS   AGE
nfd-master-gj4bh               1/1     Running   0          9m46s
nfd-master-hngrm               1/1     Running   0          9m46s
nfd-master-shwg5               1/1     Running   0          9m46s
nfd-operator-b74cbdc66-jsgqq   1/1     Running   0          10m
nfd-worker-87wpn               1/1     Running   2          9m47s
nfd-worker-d7kj8               1/1     Running   1          9m47s
nfd-worker-n4g7g                1/1     Running   1          9m47s

Reply:
@zvonkok @ahardin-rh PTAL